monai/deploy/operators/dicom_seg_writer_operator.py

```diff
@@ -157,12 +262,29 @@ def create_dicom_seg(self, image: np.ndarray, dicom_series: DICOMSeries, file_pa
         file_path.parent.absolute().mkdir(parents=True, exist_ok=True)
 
         dicom_dataset_list = [i.get_native_sop_instance() for i in dicom_series.get_sop_instances()]
-        # DICOM Seg creation
-        self._seg_writer = DICOMSegWriter()
+
         try:
-            self._seg_writer.write(image, dicom_dataset_list, str(file_path), self._seg_labels)
-            # TODO: get a class to encapsulate the seg label information.
+            version_str = get_sdk_semver()  # SDK Version
+        except Exception:
+            version_str = "0.1"  # Fall back to the initial version
+
+        seg = hd.seg.Segmentation(
+            source_images=dicom_dataset_list,
+            pixel_array=image,
+            segmentation_type=hd.seg.SegmentationTypeValues.BINARY,
+            segment_descriptions=self._seg_descs,
+            series_instance_uid=hd.UID(),
+            series_number=random_with_n_digits(4),
+            sop_instance_uid=hd.UID(),
+            instance_number=1,
+            manufacturer="The MONAI Consortium",
```

Comment:
Nit: For now this set of attributes can be left hard coded. They are mostly used for the contributing equipment module,  and have been encapsulated in the App SDK class, `EquipmentInfo`, which is still mixed in the module of the SR writer. The plan was to have the Model Info and Equipment Info all encapsulated and passed down from the App to all writer types, so that the generated DICOM objects have correct attributes conforming to the IHE AI Result Profile. profile, etc.